README.md

```diff
@@ -1,33 +1,39 @@
-# :vertical_traffic_light: RBAC service for Skygear
+# Skygear RBAC OpenAPI Specification
+[![Build Status](https://travis-ci.com/oursky/skygear-rbac.svg?branch=master)](https://travis-ci.com/oursky/skygear-rbac)
+## Steps to finish
 
-> NOTE: This service uses casbin as db name, all records e.g. policy, group are under casbin_rule table
+1. Enable [Travis](https://docs.travis-ci.com/user/getting-started/#To-get-started-with-Travis-CI%3A) for your repository (**note**: you already have `.travis.yml` file)
+1. [Create GitHub access token](https://help.github.com/articles/creating-an-access-token-for-command-line-use/); select `public_repo` on `Select scopes` section.
+1. Use the token value as a value for [Travis environment variable](https://docs.travis-ci.com/user/environment-variables/#Defining-Variables-in-Repository-Settings) with the name `GH_TOKEN`
+1. Make a test commit to trigger CI: `git commit --allow-empty -m "Test Travis CI" && git push`
+1. Wait until Travis build is finished. You can check progress by clicking on the `Build Status` badge at the top
+1. **[Optional]** You can setup [custom domain](https://help.github.com/articles/using-a-custom-domain-with-github-pages/) (just create `web/CNAME` file)
+1. **[Optional]** If your API is public consider adding it into [APIs.guru](https://APIs.guru) directory using [this form](https://apis.guru/add-api/).
+1. Delete this section ❌
```

Comment:
What is going on here. This readme is not for rbac. We should just do it not include in PR.